drivers/usb-modem/files/modules-aarch64.txt

```diff
@@ -0,0 +1,36 @@
+modules.order
+modules.builtin
+modules.builtin.modinfo
+kernel/drivers/net/usb/dm9601.ko
+kernel/drivers/net/usb/rndis_host.ko
+kernel/drivers/net/usb/smsc75xx.ko
+kernel/drivers/net/usb/sierra_net.ko
+kernel/drivers/net/usb/cdc_subset.ko
+kernel/drivers/net/usb/int51x1.ko
+kernel/drivers/net/usb/cdc_eem.ko
+kernel/drivers/net/usb/cdc_ncm.ko
+kernel/drivers/net/usb/zaurus.ko
+kernel/drivers/net/usb/net1080.ko
+kernel/drivers/net/usb/qmi_wwan.ko
+kernel/drivers/net/usb/mcs7830.ko
+kernel/drivers/net/usb/aqc111.ko
+kernel/drivers/net/usb/sr9700.ko
+kernel/drivers/net/usb/usbnet.ko
+kernel/drivers/net/usb/sr9800.ko
+kernel/drivers/net/usb/plusb.ko
+kernel/drivers/net/usb/lg-vl600.ko
+kernel/drivers/net/usb/cdc-phonet.ko
+kernel/drivers/net/usb/smsc95xx.ko
+kernel/drivers/net/usb/gl620a.ko
+kernel/drivers/net/usb/r8153_ecm.ko
+kernel/drivers/net/usb/cdc_ether.ko
+kernel/drivers/net/usb/cx82310_eth.ko
+kernel/drivers/net/usb/huawei_cdc_ncm.ko
+kernel/drivers/net/usb/kalmia.ko
+kernel/drivers/net/usb/asix.ko
+kernel/drivers/net/usb/cdc_mbim.ko
+kernel/drivers/net/usb/ax88179_178a.ko
```

Comment:
I had to fork the lists, as now `kernel/drivers/net/phy/smsc.ko` is compiled into the kernel for arm64, and I can't make it a module (some other `=y` option pulls it in as `=y`).